server/tournaments/generator-elimination.js

```diff
@@ -67,16 +67,16 @@ class Elimination {
 	}
 	getUsers(remaining) {
 		let users = [];
-		this.users.forEach((value, key) => {
+		for (const [key, value] of this.users) {
 			if (remaining && (value.isEliminated || value.isDisqualified)) return;
 			users.push(key);
-		});
+		}
 		return users;
 	}
 
 
 	generateBracket() {
-		Dex.shuffle(this.getUsers()).forEach(user => {
+		for (const user of Dex.shuffle(this.getUsers())) {
 			if (!this.tree) {
 				this.tree = {
 					tree: new TreeNode(null, {user: user}),
```

Comment:
There's a return around five lines below this line that needs to be converted to `continue`